desktopIconsIntegration.js

```diff
@@ -58,13 +58,14 @@
 import {GLib} from './dependencies/gi.js';
 import {Main} from './dependencies/shell/ui.js';
 import {ExtensionUtils} from './dependencies/shell/misc.js';
-
-import {DockManager} from './docking.js';
+import {Extension} from './dependencies/shell/extensions/extension.js';
 
 const IDENTIFIER_UUID = '130cbc66-235c-4bd6-8571-98d2d8bba5e2';
 
 export class DesktopIconsUsableAreaClass {
     constructor() {
+        const Me = Extension.lookupByURL(import.meta.url);
```

Comment:
This causes dash-to-dock to not load at all in GNOME 46:
```
(gnome-shell:21081): GNOME Shell-CRITICAL **: 13:59:17.826: Extension dash-to-dock@micxgx.gmail.com: TypeError: (intermediate value).lookupByURL is not a function

Stack trace:
  DesktopIconsUsableAreaClass@file:///home/dan/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/desktopIconsIntegration.js:67:30
  DockManager@file:///home/dan/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/docking.js:1658:40
  enable@file:///home/dan/.local/share/gnome-shell/extensions/dash-to-dock@micxgx.gmail.com/extension.js:11:23
  _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:266:38
  loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:478:32
  async*_loadExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:786:24  async*_enableAllExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:792:48
  _sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:827:20
  async*init@resource:///org/gnome/shell/ui/extensionSystem.js:76:14 
  _initializeUI@resource:///org/gnome/shell/ui/main.js:303:22
  start@resource:///org/gnome/shell/ui/main.js:175:11
  @resource:///org/gnome/shell/ui/init.js:12:47
  @resource:///org/gnome/shell/ui/init.js:21:20
```

Reply:
Yes, it seems the import form ./dependencies breaks it. The files have to be imported directly ? static function importByUrl ()

The previous MR #2132 works perfectly. That also answers Marco Trevisan's question - I forgot why I imported the files directly. I can close this MR, I just have to figure out how to re-open #2132 :smile: